includes/Dashboard.php

```diff
@@ -123,6 +123,7 @@ public function load_stories_dashboard() {
 		// TODO Preload templates.
 		$preload_paths = [
 			'/web-stories/v1/fonts',
+			'/wp/v2/web-story?context=edit&order=desc&orderby=modified&page=1&per_page=24&status=publish%2Cdraft&story_format=true',
```

Comment:
We can also preload other REST API calls here, like call published and draft stories api calls. Not sure these are required.